src/main/java/com/google/api/codegen/config/MethodConfig.java

```diff
@@ -334,10 +329,14 @@ public boolean hasRequestObjectMethod() {
   }
 
   /**
-   * Returns the Interface that should be used for the GRPC call in place of the interface
-   * in which this method appears.
+   * Returns the Interface that should be used for the GRPC call in place of the interface in which
+   * this method appears.
    */
   public String getRerouteToGrpcInterface() {
     return rerouteToGrpcInterface;
   }
+
+  static private boolean isStreamingMethod(Method method) {
```

Comment:
Ahh, I misread this - you had the `static` before the `private`; it should have been the other order.